<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameplayWindow</class>
 <widget class="QMainWindow" name="GameplayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Stop_Name_LB">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stop Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Driver_Name_Label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Driver Name:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Mute_checkBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>490</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Mute</string>
    </property>
   </widget>
   <widget class="QLabel" name="Bus_No_Label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>540</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bus No:</string>
    </property>
   </widget>
   <widget class="QLabel" name="HealthB_LB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bus's health:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PassengerStop">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>260</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Passengers at Stop:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Score_Lable">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ID_Label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ID number: </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Bus_HBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>971</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="Route_Label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Route: </string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gameView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Passenger_BusLB">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>290</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Passenger on Bus:</string>
    </property>
   </widget>
   <widget class="QLabel" name="stop_pin_but">
    <property name="geometry">
     <rect>
      <x>779</x>
      <y>390</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PPPPPPP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
